src/vs/editor/contrib/stickyScroll/browser/stickyScrollActions.ts

```diff
@@ -144,3 +144,27 @@ export class GoToStickyScrollLine extends EditorAction2 {
 		}
 	}
 }
+
+export class EscapeStickyScroll extends EditorAction2 {
+	constructor() {
+		super({
+			id: 'editor.action.escapeStickyScroll',
+			title: {
+				value: localize('escapeStickyScroll.title', "Escape Sticky Scroll"),
+				original: 'Escape Sticky Scroll'
+			},
+			precondition: ContextKeyExpr.has('config.editor.stickyScroll.enabled'),
+			keybinding: {
+				weight,
+				primary: KeyCode.Escape
+			}
+		});
+	}
+
+	runEditorCommand(_accessor: ServicesAccessor, editor: ICodeEditor) {
+		const stickyScrollController = StickyScrollController.get(editor);
+		if (stickyScrollController) {
+			stickyScrollController.escape();
+		}
```

Comment:
```suggestion
		StickyScrollController.get(editor)?.escape();
```

Reply:
Even better though would be to use the editor service to focus the active editor, that saves you from exposing an escape function on the controller

Reply:
Just tried to access the editor service, it would appear importing it into the stickyScrollAction.ts causes an import violation error:

<img width="845" alt="Screenshot 2023-03-22 at 16 30 39" src="https://user-images.githubusercontent.com/61460952/226956007-fbbc927d-015d-4897-8795-0472c8154489.png">

Reply:
Ah ok, .selectEditor works then